encoder/vaapiencoder_vp9.cpp

```diff
@@ -278,7 +278,9 @@ bool VaapiEncoderVP9::fill(VAEncPictureParameterBufferVP9* picParam,
 
     picParam->pic_flags.bits.show_frame = 1;
 
-    picParam->luma_ac_qindex = kDefaultQPValue;
+    picParam->luma_ac_qindex =
+        (initQP() > minQP() && initQP() < maxQP()) ? initQP() : kDefaultQPValue;
```

Comment:
thanks @xuguangxin , You're right.
In that case, we can set minimum and maximum.